test/forkMainnet/AMMQuoter.t.sol

```diff
@@ -45,16 +45,27 @@ contract AMMQuoterTest is StrategySharedSetup {
         vm.label(address(this), "TestingContract");
         vm.label(address(ammQuoter), "AMMQuoterContract");
         vm.label(UNISWAP_V2_ADDRESS, "UniswapV2");
-        vm.label(SUSHISWAP_ADDRESS, "Sushiswap");
         vm.label(UNISWAP_V3_ADDRESS, "UniswapV3");
         vm.label(UNISWAP_V3_QUOTER_ADDRESS, "UniswapV3Quoter");
+        vm.label(SUSHISWAP_ADDRESS, "Sushiswap");
+        vm.label(BALANCER_V2_ADDRESS, "Balancer");
     }
 
     /*********************************
      *          Test: setup          *
      *********************************/
 
     function testSetupAMMQuoter() public {
+        // Check fork mainnet addresses are not zero addresses
+        assertTrue(uint160(UNISWAP_V2_ADDRESS) != 0);
```

Comment:
Q: Couldn't it compare to `address(0)` like `UNISWAP_V2_ADDRESS != address(0)`?

Reply:
Somehow I mistaken an error as not able to compare but in fact it can. 🤦‍♂️